.github/workflows/job-image.yml

```diff
@@ -0,0 +1,51 @@
+name: job-image
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'releng/**'
+      - 'charts/**'
+      - 'manifests/**'
+      - 'k8s-config/**'
+      - 'docs/**'
+      - '*.md$'
+  pull_request:
+    paths-ignore:
+      - 'releng/**'
+      - 'charts/**'
+      - 'manifests/**'
+      - 'k8s-config/**'
+      - 'docs/**'
+      - '*.md$'
+
+jobs:
+  execute-generate:
+    runs-on: ubuntu-latest
+    name: job-image
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.GH_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.GH_AWS_SECRET_ACCESS_KEY }}
+      AWS_EC2_METADATA_DISABLED: true
+      DEV_REGISTRY: ${{ secrets.DEV_REGISTRY }}
```

Comment:
non blocker but is there a way to do env variables set in github that aren't secrets? cuz we don't actually care about exposing this value, but we want to set it at the github repo level if that makes sense (circle didn't have a way to do this)

Reply:
I'll look things over - but I think that there are only secrets same as CircleCI.